lib/jekyll/commands/doctor.rb

```diff
@@ -35,7 +37,8 @@ def healthy?(site)
             fsnotify_buggy?(site),
             !deprecated_relative_permalinks(site),
             !conflicting_urls(site),
-            !urls_only_differ_by_case(site)
+            !urls_only_differ_by_case(site),
+            proper_site_url?(site)
```

Comment:
@pathawks what prevents us from adding `Style/TrailingCommaInLiteral: { EnforcedStyleForMultiline: consistent_comma }`to `.rubocop.yml`?